parity/params.rs

```diff
@@ -73,11 +74,12 @@ impl str::FromStr for SpecType {
 			"ellaism" => SpecType::Ellaism,
 			"mix" => SpecType::Mix,
 			"callisto" => SpecType::Callisto,
-			"morden" | "classic-testnet" => SpecType::Morden,
+			"morden" => SpecType::Morden,
+			"mordor" | "classic-testnet" => SpecType::Morden,
 			"ropsten" => SpecType::Ropsten,
-			"kovan" | "testnet" => SpecType::Kovan,
+			"kovan" => SpecType::Kovan,
 			"rinkeby" => SpecType::Rinkeby,
-			"goerli" | "görli" => SpecType::Goerli,
+			"goerli" | "görli" | "testnet" => SpecType::Goerli,
```

Comment:
Hmm, how about we remove "testnet" altogether? What is it useful for?

Reply:
Sorry, I missed the explanation in the PR descr. Something for the relnotes then (cc @s3krit)

Reply:
it's a legacy option. I'm open to removing it.

Reply:
Made a note to myself for the release notes. Thanks